docs/stdlib/doc/env.md

```diff
@@ -0,0 +1,202 @@
+## `color_echo`
+```ab
+pub fun color_echo(message: Text, color: Num): Null 
+```
+
+Prints a text with a specified color.
+
+
+## `confirm`
+```ab
+pub fun confirm(prompt: Text, default_yes: Bool = false): Bool 
+```
+
+Creates a confirm prompt (Yes/No), and returns true if the choice is Yes.
+
+"No" is the default choice, set default_yes to true for "Yes" as default choice.
+
+
+## `echo_info`
+```ab
+pub fun echo_info(message: Text): Null 
+```
+
+Prints a text as a info message.
+
+
+## `echo_success`
+```ab
+pub fun echo_success(message: Text): Null 
+```
+
+Prints a text as a success message.
+
+
+## `echo_warning`
+```ab
+pub fun echo_warning(message: Text): Null 
+```
+
+Prints a text as a warning message.
+
+
+## `error`
+```ab
+pub fun error(message: Text, exit_code: Num = 1): Null 
+```
+
+Prints a text as a error and exits if the status code is greater than 0.
+
+
+## `exit`
+```ab
+pub fun exit(code: Num): Null 
+```
+
+Closes the script.
```

Comment:
"closes" the script? oh come on. that is not even a close explanation to what it does